tests/integration/targets/inventory_aws_mq/requirements.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We should not need this file if I am not mistaken.

Reply:
correct, I used this when I was testing against amazon.aws collection. fixed